easybuild/easyconfigs/h/HOMER/HOMER-4.11.1-foss-2022b.eb

```diff
@@ -0,0 +1,53 @@
+# This installs the entire suite, to install only base Homer package run :
+# cd %(installdir)s; perl ./configureHomer.pl -install homer
+
+easyblock = 'CmdCp'
+
+name = 'HOMER'
+version = '4.11.1'
+
+homepage = 'http://homer.ucsd.edu/homer/'
+description = """HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif
+Discovery and ChIP-Seq analysis. It is a collection of command line programs for unix-style operating
+systems written in mostly perl and c++. Homer was primarily written as a de novo motif discovery
+algorithm that is well suited for finding 8-12 bp motifs in large scale genomics data."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Perl', '5.36.0'),
+    ('zlib', '1.2.12'),
+    ('Zip', '3.0'),
+    ('weblogo', '2.8.2'),
+    ('SAMtools', '1.17'),
+    ('Kent_tools', '461'),
+    ('R', '4.2.2'),
+    ('R-bundle-Bioconductor', '3.16', '-R-4.2.2')
+]
+
+source_urls = ['http://homer.ucsd.edu/homer/data/software']
+sources = ['homer.v%(version)s.zip']
+checksums = ['80d1cd00616729894017b24a36a2ef81f9cde8bd364e875aead1e0cfb500c82b']
+
+unpack_options = " && cp %(builddir)s/configureHomer.pl %(installdir)s "
+start_dir = "%(installdir)s"
+cmds_map = [('', 'perl ./configureHomer.pl -install -all')]
+skipsteps = ['install']
+
+files_to_copy = [
+    'bin', 'config.txt', 'data',
+    'DoughnutDocumentation.pdf',
+    'motifs',
+    'update',
+]
```

Comment:
this is a very nonconventional and rather misleading way. i suggest the following instead:
- use easyblock `PackedBinary`
- replace the selected text with the following:
```
buildininstalldir = True
skipsteps = ['install']
postinstallcmds = ['perl ./configureHomer.pl -install -all -keepScript']
```
note that i've added option `-keepScript` to avoid that the script will update itself and thus effectively installs another version of HOMER than the one originally downloaded. let me know if you disagree.